app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -204,18 +226,28 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 			emptyFlow(), emptyFlow(),
 			sunroof
 	)
+	private val ACFields: List<Flow<String>> = listOf(
+			tempExterior, tempInterior,
+			tempEvaporator, tempExchanger,
+			flowOf(L.CARINFO_COMPRESSOR),emptyFlow(),
+			ACCompressorState, emptyFlow(),
+			ACCompressorLevel,
+	)
 
 	val basicCategories = LinkedHashMap<String, List<Flow<String>>>().apply {
 		put(L.CARINFO_TITLE, overviewFields)
 		put(L.CARINFO_TITLE_DRIVING, drivingFields)
+		put (L.CARINFO_TITLE_PERFORMANCE, drivingPerformanceFields)
 	}
 	val advancedCategories = LinkedHashMap<String, List<Flow<String>>>().apply {
 		put(L.CARINFO_TITLE, overviewFields)
 		put(L.CARINFO_TITLE_DRIVING + " ", drivingAdvancedFields)   // slightly different key for the allCategories
 
 		// add more pages like this:
-		put("GPS", gpsFields)
-		put("Window Status", windowFields)
+		put (L.CARINFO_TITLE_PERFORMANCE + " ", drivingPerformanceFields)
```

Comment:
The "slightly different key" hack shouldn't matter for this, since it's the same `drivingPerformanceFields` in both lists.
Could you also remove the space in `put (`?

Reply:
Ooops...forgot to remove it. Done.